config/opal_check_openfabrics.m4

```diff
@@ -301,12 +301,13 @@ AC_DEFUN([OPAL_CHECK_OPENFABRICS],[
 
 AC_DEFUN([OPAL_CHECK_OPENFABRICS_CM_ARGS],[
     #
-    # ConnectX XRC support
+    # ConnectX XRC support - disabled see issue #3890
     #
-    AC_ARG_ENABLE([openib-connectx-xrc],
-        [AC_HELP_STRING([--enable-openib-connectx-xrc],
-                        [Enable ConnectX XRC support in the openib BTL. If you do not have InfiniBand ConnectX adapters, you may disable the ConnectX XRC support. If you do not know which InfiniBand adapter is installed on your cluster, leave this option enabled (default: enabled)])],
-                        [enable_connectx_xrc="$enableval"], [enable_connectx_xrc="yes"])
+dnl    AC_ARG_ENABLE([openib-connectx-xrc],
```

Comment:
If you're not going to remove this code, you should probably put a comment here as to why you're commenting this out.  Perhaps even refer to https://github.com/open-mpi/ompi/issues/3890 and/or https://github.com/open-mpi/ompi/issues/3969.